docs/howtos/vqe_ucc.rst

```diff
@@ -0,0 +1,102 @@
+.. _how-to-vqe-ucc:
+
+How to use a UCC-like ansatz with a ``VQE``
+===========================================
+
+When using a :class:`~qiskit_nature.second_q.circuit.library.UCC`-style ansatz with a
+:class:`~qiskit.algorithms.minimum_eigensolvers.VQE` one needs to pay particular attention to the
+:attr:`~qiskit.algorithms.minimum_eigensolvers.VQE.initial_point` attribute which indicates from
+which set of initial parameters the optimization routine should start.
+By default, the algorithm will start from a *random* initial point. In this how to we show how one
```

Comment:
```suggestion
By default, VQE will start from a *random* initial point. In this how to we show how one
```
I am wondering if saying "the algorithm" is a bit vague here, given this also uses algorithms from nature - maybe explictly stating  "VQE" is clearer.